pkgs/tools/misc/mcfly/default.nix

```diff
@@ -2,30 +2,21 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "mcfly";
-  version = "0.5.8";
+  version = "0.5.9";
 
   src = fetchFromGitHub {
     owner = "cantino";
     repo = "mcfly";
     rev = "v${version}";
-    sha256 = "sha256-D8ScF/3qyT0VuMGmWkkeGRyCu4LdOXt1wvR9whbNURE=";
+    sha256 = "0i3qjgq1b8h3bzc7rxa60kq1yc2im9m6dgzrvial086a1zk8s81r";
   };
 
-  postInstall = ''
-    substituteInPlace mcfly.bash --replace '$(which mcfly)' $out/bin/mcfly
-    substituteInPlace mcfly.zsh  --replace '$(which mcfly)' $out/bin/mcfly
-    substituteInPlace mcfly.fish --replace '(which mcfly)' $out/bin/mcfly
-    install -Dm644 -t $out/share/mcfly mcfly.bash
```

Comment:
I couldn't find in the release where those were not needed anymore.
Where those not needed from the begining ?
I looked upstream and those files are still there.
Does mcfly work properly without those ?

Reply:
these are no longer needed, mcfly switched to the `init` subcommand since mcfly 0.5.4 (released on Feb 28, 2021)
https://github.com/cantino/mcfly#installing-using-our-install-script

Reply:
Got it, some other program like home-manager will take care of the init https://github.com/nix-community/home-manager/blob/master/modules/programs/mcfly.nix#L76

Reply:
Looks good to me then.